modules/appeals_api/config/schemas/supplemental_claims/v0/200995_headers.json

```diff
@@ -6,130 +6,14 @@
-      },
-      "required": [
-        "X-VA-SSN",
-        "X-VA-First-Name",
-        "X-VA-Last-Name",
-        "X-VA-Birth-Date",
-        "X-VA-ICN"
-      ]
-    },
-
-    "X-VA-SSN": {
-      "type": "string",
-      "description": "SSN",
-      "pattern": "^[0-9]{9}",
-      "minLength": 9,
-      "maxLength": 9
-    },
-
-    "X-VA-ICN": {
-      "type": "string",
-      "description": "Veteran's ICN",
-      "pattern": "^[0-9]{10}V[0-9]{6}$",
-      "minLength": 17,
-      "maxLength": 17
-    },
-
-    "X-VA-First-Name": {
-      "allOf": [
-        { "description": "Veteran's first name", "minLength": 1, "maxLength": 30 },
-        { "$ref": "nonBlankString.json" },
-        { "$comment":  "can be whitespace, to accommodate those with 1 legal name" }
-      ]
-    },
-
-    "X-VA-Middle-Initial": {
-      "allOf": [
-        { "description": "Veteran's middle initial", "minLength": 1, "maxLength": 1 },
-        { "$ref": "nonBlankString.json" }
-      ]
-    },
-
-    "X-VA-Last-Name": {
-      "allOf": [
-        { "description":  "Veteran's last name", "minLength": 1, "maxLength": 40 },
-        { "$ref":  "nonBlankString.json" }
-      ]
-    },
-
-    "X-VA-Birth-Date": {
-      "description": "Veteran's birth date",
-      "type": "string", "format": "date"
-    },
-
-    "X-VA-NonVeteranClaimant-First-Name": {
-      "allOf": [
-        { "description": "Non-Veteran Claimant's first name", "minLength": 1, "maxLength": 30 },
-        { "$ref": "nonBlankString.json" },
-        { "$comment":  "can be whitespace, to accommodate those with 1 legal name" }
-      ]
-    },
-
-    "X-VA-NonVeteranClaimant-Middle-Initial": {
-      "allOf": [
-        { "description": "Non-Veteran Claimant's middle initial", "minLength": 1, "maxLength": 1 },
-        { "$ref": "nonBlankString.json" }
-      ]
-    },
-
-    "X-VA-NonVeteranClaimant-Last-Name": {
-      "allOf": [
-        { "description":  "Non-Veteran's Claimaint's last name", "minLength": 1, "maxLength": 40 },
-        { "$ref":  "nonBlankString.json" }
-      ]
-    },
-
-    "X-VA-File-Number": {
-      "allOf": [
-        { "description": "Veteran's VA file number (c-file / css)", "minLength": 1, "maxLength": 9 },
-        { "$ref": "nonBlankString.json" }
-      ]
-    },
-
-    "X-VA-Service-Number": {
-      "allOf": [
-        { "description":  "Veteran's service number", "minLength":  1, "maxLength":  9 },
-        { "$ref":  "nonBlankString.json" }
-      ]
-    },
-
-    "X-VA-Insurance-Policy-Number": {
-      "allOf": [
-        { "description": "Veteran's insurance policy number", "minLength": 1, "maxLength":  18 },
-        { "$ref": "nonBlankString.json" }
-      ]
+        "X-Consumer-ID":        { "$ref": "#/definitions/X-Consumer-ID"}
+      }
     },
 
     "X-Consumer-Username": {
       "allOf": [
-        { "description": "Consumer User Name (passed from Kong)", "minLength": 1, "maxLength": 255 },
```

Comment:
It seemed odd to put `(passed from Kong)` in our user-facing docs, so I updated this description